unlock-app/src/__tests__/components/lock/Overlay.test.js

```diff
@@ -61,18 +61,36 @@ describe('Overlay', () => {
   describe('displayError', () => {
     it('should display children if there is no error', () => {
       expect.assertions(1)
-      const wrapper = rtl.render(displayError(false, {}, <div>children</div>))
+      const wrapper = rtl.render(
+        displayError(true)(false, {}, <div>children</div>)
```

Comment:
Sometimes when the value is not very obvious (often times for boolean...) I do this :

```suggestion
displayError(true /* isMainWindow */)(false, {}, <div>children</div>)
```